packages/iov-bns/src/bnsconnection.spec.ts

```diff
@@ -1615,6 +1442,7 @@ describe("BnsConnection", () => {
     expect(faucetEndBalance.quantity).toEqual(
       Long.fromString(faucetStartBalance.quantity)
         .subtract(68_000000000)
+        .subtract(10_000000) // the fee (0.01 CASH)
```

Comment:
What about `0_010000000` to use _ for the full/fractional separation like above?

Reply:
Sure